internal/cfgplugins/interface.go

```diff
@@ -725,3 +726,28 @@ func DeleteAggregate(t *testing.T, dut *ondatra.DUTDevice, aggID string, dutAggP
 		gnmi.Delete(t, dut, gnmi.OC().Interface(port.Name()).Ethernet().AggregateId().Config())
 	}
 }
+
+// SetupStaticAggregateAtomically sets up the static aggregate interface atomically.
+func SetupStaticAggregateAtomically(t *testing.T, dut *ondatra.DUTDevice, aggPorts []*ondatra.Port, aggID string, dutLag attrs.Attributes) {
```

Comment:
The interface config parameters should be in a struct.  For example:
https://github.com/openconfig/featureprofiles/blob/7ce57f30158e82e164f0322ec34c3cd353dad773/internal/cfgplugins/sflow.go#L43

See the [README for cfgplugins](https://github.com/openconfig/featureprofiles/blob/main/internal/cfgplugins/README.md)